### PR TITLE
Tweak communication guidelines

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: GIT_AND_GITHUB.md [hunk 1]
```diff
@@ -82,22 +82,22 @@ on something you've learned, and so on. The goal here is to make the whole team
 in the process.
 
 Keep in mind that you are reviewing something that someone else has worked on really 
-hard, so always be respectful. The guidelines from GitHub for giving feedback are a 
+hard, so always be respectful. These guidelines for for giving feedback are a
```
Comment (elland) [inline GIT_AND_GITHUB.md]: "for for"

File: GIT_AND_GITHUB.md [hunk 2]
```diff
@@ -82,22 +82,22 @@ on something you've learned, and so on. The goal here is to make the whole team
...
-[Source: How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)
+- Familiarise yourself with the context of the issue.
+- If you disagree strongly, consider giving it a few minutes before responding; think before you react. Try to empathise.
+- Ask, don’t tell. (“What do you think about trying…?” rather than “Don’t do…”)
+- Explain your reasons why code should be changed. (Not in line with the style guide? A personal preference?)
+- Offer ways to simplify or improve code.
+- Avoid using derogatory terms, like “stupid”, when referring to the work someone has produced. Don’t shame, no one started out knowing.
+- Be humble. (“I’m not sure, let’s try…”)
+- Never deal in absolutes. Unless needed. (“We should never store sensitive data in plain text.”)
+- Aim to develop professional skills, group knowledge and product quality, through group critique. That goes both way: try to be open and welcoming to criticism.
```
Comment (elland) [inline GIT_AND_GITHUB.md:96]: "both way**s**"

**Comment by 3lvis** [comment]
I basically moved the ones you already discussed on https://github.com/bakkenbaeck/iOS-playbook/pull/76. Since no changes were done, I'll just merge.

File: GIT_AND_GITHUB.md [hunk 3]
```diff
@@ -82,22 +82,22 @@ on something you've learned, and so on. The goal here is to make the whole team
...
+- Be aware of negative bias with online communication. (If content is neutral, we assume the tone is negative.) Try to use positive language instead.
```
Comment (elland) [inline GIT_AND_GITHUB.md:97]: "Try using positive"

**Review by elland — COMMENTED** [review]
👌

**Comment by elland** [comment]
Maybe I should have made the comments back at the main PR, then. 😄

**Comment by 3lvis** [comment]
@Elland Now is your chance.

**Comment by elland** [comment]
👯

**Pull request merged**